src/components/users/EditProfileForm.js

```diff
@@ -163,6 +182,7 @@ const EditProfileForm = ({ user, setEditProfile, newImg }) => {
 EditProfileForm.propTypes = {
   user: userShape,
   setEditProfile: func,
+  newImg: object,
```

Comment:
is this an object? 🤔

Reply:
I'm not quite sure the type of this object. Let me check

Reply:
Anyway, to use object type itself in propTypes is pretty useless. The idea behind to use propTypes is that you got warned each time a component prop is not what it's supposed to be. If you use object per se, whatever object this component gets will be ok, if I pass down { string: 'hola' } would be ok, same for { a: {b: 3 }}. Make sure to give an accurate shape to the propTypes so you will be really taking advantage of them. :)

Reply:
Yes is an object. A File object